docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app

```

Comment:
Mounting the entire current directory into the container can lead to potential issues with file permissions and unnecessary files being included. Consider using a `.dockerignore` file to exclude unnecessary files or specifying only the directories needed for the application.